### DISCUSSION TITLE
alist拒绝连接

### DISCUSSION THREAD

**Discussion by Jasonanywhere** [AlistGo/alist#4570] [General]
报错信息如下：
ERROR RESTY Post "https://api.nn.ci/alist/ali_open/token": dial tcp 45.66.130.152:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond., Attempt 2
如何解决

**Comment by leonevil1412** [comment]
我也是

**Comment by MeTerminator** [comment]
我也是，必须挂梯子才能用

**Comment by MrL646** [comment]
试试用http
http://api.nn.ci/alist/ali_open/token

**Comment by xhofe** [comment]
- #4572
